src/core/config.test.ts

```diff
@@ -49,6 +49,6 @@ describe("useLegacyTemplateEngine", () => {
             model: { text: "with data" }
         });
 
-        expect(renderedTemplate.innerHTML).toBe("Template with data");
+        expect(renderedTemplate.innerHTML).toBe(`<div>Template with data</div><span style=\"display: none;\"></span>`);
```

Comment:
The changes are going to be a BC, right?
Templates in existing projects will be wrapped in a div

Reply:
Discussed in private - this should be OK, as DevExtreme treats templates in diffrent ways